cdap-docs/_common/_source/index.rst

```diff
@@ -14,20 +14,20 @@ CDAP Documentation v\ |version|
 
 .. rubric:: Introduction to the Cask Data Application Platform
 
-The Cask |(TM)| Data Application Platform (CDAP) is an open source application development
+The Cask |(TM)| Data Application Platform (CDAP) is an integrated, open source application development
 platform for the Hadoop ecosystem that provides developers with data and application
-virtualization to accelerate application development, address a broader range of real-time
+abstractions to accelerate application development, address a broader range of real-time
```

Comment:
perhaps also mention "to simplify"?: abstractions to simplify and accelerate app development